### DISCUSSION TITLE
PHP 7.3 и Maxsite

### DISCUSSION THREAD

**Discussion by Sigoy** [maxsite/cms#456] [Q&A]
Максим,почему макси не работает с PHP 7.3?

**Comment by maxsite** [comment]
Что именно не работает?

**Reply by Sigoy** [reply]
Все кроме Главной.Типа ненайден файл.Откат на php7.2 снял проблему. Ставил версию MaxSite CMS 8.

**Reply by Sigoy** [reply]
Снова переключил Laragon на php7.3 и проблем нет.
